CRM/Export/Form/Select.php

```diff
@@ -231,7 +231,7 @@ public function buildQuickForm() {
     $exportOptions = $mergeOptions = $postalMailing = array();
     $exportOptions[] = $this->createElement('radio',
       NULL, NULL,
-      ts('Export PRIMARY fields'),
+      ts('Export %1 fields', array(1 => empty($this->_customSearchID) ? 'PRIMARY' : 'custom search')),
```

Comment:
The replacement for `%1` is an untranslated English string, so I think it would be better to do:

    empty($this->_customSearchID) ? ts('Export PRIMARY fields') : ts('Export custom search fields'),